package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "NovelCOVID-API",
+  "name": "novelcovid-api",
```

Comment:
Why are we changing this?

Reply:
this was changed long ago, but the package-lock.json hasnt been updated since then. its to be conform with package.json naming rules.